modules/packages/LinearAlgebra.chpl

```diff
@@ -2064,6 +2326,114 @@ proc kron(A: [?ADom] ?eltType, B: [?BDom] eltType) {
 }
 
 
+/*
+  Matrix exponential using Pade approximation
```

Comment:
This comment I will be a bit more strict on following the documentation style requirements